rustler_bigint/README.md

```diff
@@ -0,0 +1,34 @@
+# Rustler BigInt
+
```

Comment:
maybe add a link to the docs? `https://docs.rs/rustler_bigint/latest/rustler_bigint/` (or ofcourse the real url when we know what it is :P )